packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx

```diff
@@ -42,14 +42,16 @@ import { FilterComparator } from '@ovh-ux/manager-core-api';
 import { Spinner } from '@/components/spinner/Spinner.component';
 import { TInstance, useInstances } from '@/data/hooks/instances/useInstances';
 import StatusChip from '@/components/statusChip/StatusChip.component';
+import { Breadcrumb } from '@/components/breadcrumb/Breadcrumb.component';
 
 const initialSorting = {
   id: 'name',
   desc: false,
 };
 
 const Instances: FC = () => {
-  const { t } = useTranslation('list');
+  const { t: tList } = useTranslation('list');
+  const { t: tCommon } = useTranslation('common');
```

Comment:
Why not use https://react.i18next.com/guides/multiple-translation-files

Reply:
Yep, agree 👍🏻